tests/dataloader/test_language_generation.py

```
@@ -43,6 +45,28 @@ def base_test_init(self, dl):
 		# assert the data has invalid token
 		assert dl.all_vocab_size > dl.vocab_size
 
+
+		try:
+			gen = Dataloader().get_all_subclasses()
+			for each in gen:
+				pass
+			Dataloader().load_class('LanguageGeneration')
+			Dataloader().load_class('None')
+		except Exception:
```

You don't need to catch the exception. Just let it raise.